portal-bridge/src/types/mode.rs

```diff
@@ -77,7 +77,7 @@ impl FromStr for BridgeMode {
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         match s {
             "latest" => Ok(BridgeMode::Latest),
-            "fourfours" => Ok(BridgeMode::FourFours),
+            "fourfours" => Ok(BridgeMode::FourFours(FourFoursModeType::Random)),
```

Comment:
It's worth adding a couple test cases to the test below to test this & the `fourfours:e*` use case